### PR TITLE
Fixes for Rhino installers with Rhino8 intro

### PR DESCRIPTION
**Pull request by adamhathcock**
Extension is wrong for Rhino 7 and 8 should use common stuff for Rhino

### CODE REVIEW

**Review by oguzhankoral — CHANGES_REQUESTED** [review]

File: Connectors/Rhino/Speckle.Connectors.Rhino8/Speckle.Connectors.Rhino8.csproj [hunk 1]
```diff
@@ -2,11 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>net48</TargetFramework>
-    <TargetExt>.rhp</TargetExt>
```
[the remark this answers is not in the document]
Comment (adamhathcock) [reply Connectors/Rhino/Speckle.Connectors.Rhino8/Speckle.Connectors.Rhino8.csproj:5]: It's not removed, it's part of the targets file that `RhinoVersion` should add

[the remark this answers is not in the document]
Comment (oguzhankoral) [reply Connectors/Rhino/Speckle.Connectors.Rhino8/Speckle.Connectors.Rhino8.csproj:5]: ok then why it is not working?

[the remark this answers is not in the document]
Comment (adamhathcock) [reply Connectors/Rhino/Speckle.Connectors.Rhino8/Speckle.Connectors.Rhino8.csproj:5]: it was just a problem in the launch json.  Fixed that

**Review by oguzhankoral — APPROVED** [review]
Tested both 7 and 8. works!

**Pull request merged**